### PR TITLE
TST: update unit tests for pysat 3.0.0

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

In compliance with NEP 029, support is maintained for python versions released in the last 42 months.  Pysat 3.0.0 will drop support for python 2.7, making the minimum supported version python 3.6.  With the release of python 3.8, updating travis to support 3.6 and 3.8 in the automated unit tests.

## Type of change

- Testing Update (non-breaking change which fixes an issue)

# How Has This Been Tested?

Change in Travis setup, so tested there.

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``master``) branch
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,10 +32,6 @@ install:
 before_script:
   # set up display screen
   - export DISPLAY=:99.0
-  - if [[ $TRAVIS_PYTHON_VERSION < "3.7" ]]; then
-      sh -e /etc/init.d/xvfb start;
-      sleep 3;
-    fi
```
Comment (jklenzing) [inline .travis.yml]: This bit was only needed for python 2.7

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,10 @@
 language: python
 matrix:
   include:
-    - python: 2.7
+    - python: 3.6
+      dist: xenial
       services: xvfb
-    - python: 3.7
+    - python: 3.8
```
Comment (aburrell) [inline .travis.yml]: Should we not continue to test 3.7 as well?

Comment (jklenzing) [reply .travis.yml]: We can continue to test all versions.  I'll run a test to see what that translates to in run time

Comment (jklenzing) [reply .travis.yml]: All three are running.  We can't run more than two parallel tasks, so total run time (six tests) for a pull request is ~1 hr 12 min.

Comment (jklenzing) [reply .travis.yml]: The python 3.8 tests take an extra five minutes or so.  I suspect this is because it is not auto-installed on travis yet.

**Pull request merged**